src/PortlandLabs/ConcreteCmsTheme/Navigation/HeaderNavigationFactory.php

```diff
@@ -23,33 +23,65 @@ class HeaderNavigationFactory implements ApplicationAwareInterface, NavigationFa
     public function createNavigation(): NavigationInterface
     {
         $activeSection = $this->activeSection;
+        // Determine if this navigation menu is for the concretecms.org site or not
+        $siteTarget = $_ENV['URL_SITE_MARKETING_OPENSOURCE'];
+        $siteDetected = \Core::make('site')->getSite()->getSiteCanonicalURL();
+        // remove trailing slash if exists
+        if (strcmp('/', substr($siteDetected, -1)) === 0) {
+            $siteDetected = substr($siteDetected, 0, -1);
```

Comment:
You could remove this conditional by doing `$siteDetected = rtrim(\Core::make('site')->getSite()->getSiteCanonicalURL(), '/');` with the same effect (though rtrim will remove all trailing slashes so `foo////` becomes `foo` instead of `foo///`